hydromt/_typing/type_def.py

```diff
@@ -95,3 +98,4 @@ def _validate_bbox(
 GeomBuffer = int
 
 ModeLike = Union[ModelMode, str]
+PandasLike = Union[pd.DataFrame, pd.Series]
```

Comment:
I know it avoids repetition but not sure it makes the api definition of the function and accepted argument types clearer to users in the end

Reply:
I can see what you mean, in this case it might not be worth it since pandas is a relatively small bit of the code base, so I'll revert this.